addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/dialogs/ZestRecordScriptDialog.java

```diff
@@ -176,6 +186,17 @@ private List<String> getSites() {
         return list;
     }
 
+    private void launchBrowser(String url, String browserName) {
+        ExtensionSelenium extSelenium =
+                Control.getSingleton().getExtensionLoader().getExtension(ExtensionSelenium.class);
+        WebDriver wd = extSelenium.getProxiedBrowserByName(browserName);
+        String zapurl = wd.getCurrentUrl();
```

Comment:
The browsers are opened without a URL set, for Firefox this will return `about:blank`.

Reply:
No, because of this redirect script it goes to the callback URL through which I am extracting the callback URL and configuring it into the browser extension. 
https://github.com/zaproxy/zap-extensions/blob/main/addOns/client/src/main/java/org/zaproxy/addon/client/RedirectScript.java#L34C3-L36

Reply:
This case: https://github.com/zaproxy/zap-extensions/pull/4730#issuecomment-1640396498

Reply:
Bump, though I'm fine if this is handled in another PR.